src/components/InputWithLabel/index.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react'
+import Input from '@components/Input'
+import { InputWrapper, StyledLabel } from './styles'
+
+type Props = {
+  id: string,
+  label: string
+}
+
+const InputWithLabel = (props: Props) => {
+  const { id, label } = props
+  return (
```

Comment:
Did you commit with `--no-verify`? There should be an empty line before `return`s added automatically 🤔

Reply:
I definitely not pushed with `--no-verify`
It seems the ESLint/Prettier/precommit configuration is somewhat not working correctly...